Scotty.sh

```
@@ -0,0 +1,90 @@
+startDeployment() {
+    git checkout $BRANCH
+    git pull origin $BRANCH
+}
+
+# @task on:remote
+cloneRepository() {
+    [ -d $RELEASES_DIR ] || mkdir -p $RELEASES_DIR
+    [ -d $PERSISTENT_DIR ] || mkdir -p $PERSISTENT_DIR
+    [ -d $PERSISTENT_DIR/storage ] || mkdir -p $PERSISTENT_DIR/storage
+
+    cd $RELEASES_DIR
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)